proprietary/tokens/src/common/ams/link-appearance.tokens.json

```diff
@@ -1,11 +1,11 @@
 {
   "ams": {
     "link-appearance": {
-      "color": { "value": "{ams.color.primary-blue}" },
+      "color": { "value": "{ams.brand.color.blue.60}" },
```

Comment:
And if you maintain this you don't need to prefix common tokens. The prefixing is also going to cause a lot of work in Figma.

Reply:
For example:
- You add a sementic colour called "Interaction default" in common and you link it to Blue 40
- Your button, links etc get a token referring to "Interaction Default"
- Now design changes the Interaction Default to Blue 60

In your setup you'll have to refactor all your components instead of just changing the semantic colour link in common.

Reply:
https://www.figma.com/blog/the-future-of-design-systems-is-semantic/#:~:text=When%20a%20variable%20acting%20as%20a%20global%20token%20is%20updated%2C%20then%20everything%20downstream%20changes%2C%20too.

Reply:
Yes, I agree, completely. Having this middle layer of semantic of common tokens has always been the plan. We have a few already, among which the link appearance that you’re commenting on, funny enough :) Did you see my suggestion for the [setup of common colour tokens](https://teams.microsoft.com/l/message/19:11603d9b98aa4ba9984a2bcba4f0c735@thread.tacv2/1731669613284?tenantId=72fca1b1-2c2e-4376-a445-294d80196804&groupId=381b5f11-b342-4a3a-8a78-8b371a90457d&parentMessageId=1731665737282&teamName=Amsterdam%20Design%20System&channelName=Team&createdTime=1731669613284&ngc=true) in Teams?

But just not in this PR. Introducing the shades and the level system is a big enough change in itself.

The only functional change I have done is to ensure all disabled colours use neutral-60, which itself is a step towards `ams.common.color.disabled`.

Let’s merge this and we can implement the rest of the plans for color tokens, of which common tokens is one.